enterprise/server/util/oci/oci_test.go

```diff
@@ -510,6 +509,8 @@ func layerFiles(t *testing.T, layer v1.Layer) map[string][]byte {
 			contents[header.Name] = buf.Bytes()
 		}
 	}
+	err = rc.Close()
+	require.NoError(t, err)
```

Comment:
I would still `defer` the close but `defer func() { err := rc.Close(); require.NoError(t, err) }()`, so that if the `require` fails above then the test still cleans up after itself.